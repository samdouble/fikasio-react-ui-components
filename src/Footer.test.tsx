import React from 'react';
import renderer from 'react-test-renderer';
import { Footer } from '.';

describe('Footer', () => {
  it('Renders correctly', () => {
    const componentJSON = renderer.create(<Footer />).toJSON();
    expect(componentJSON).toMatchSnapshot();
  });
});
