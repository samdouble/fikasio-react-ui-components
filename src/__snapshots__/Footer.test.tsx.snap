// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Renders correctly 1`] = `
<div
  className="ii-loader ii-theme-dark"
  style={Object {}}
>
  <div
    className="ii-loader-ring"
  >
    <div />
    <div />
    <div />
    <div />
  </div>
</div>
`;
